src/framework/parsers/ply.js

```diff
@@ -591,7 +591,7 @@ class PlyParser {
      */
     async load(url, callback, asset) {
         try {
-            const response = await fetch(url.load);
+            const response = await (asset.file?.contents ?? fetch(url.load));
```

Comment:
at least add a comment, if not the docs

Reply:
I'm not sure it's ideal to add a private way that an external projects depend on though.

Reply:
I can document it rn if you prefer?

Reply:
(comment added)

Reply:
Some assets already support `file.contents` btw, this change isn't a huge departure.

See https://github.com/playcanvas/engine/blob/main/src/framework/asset/asset.js#L250.

Reply:
It'd be great to document if we think this is an API we want to have. We're putting plenty of effort to remove undocumented things that project depend on, so lets not add another one.

Reply:
I have tested this locally and the approach seems to work very well.

But I can't test and profile on DEV and PROD with caching et al and on iOS devices until it's available.

So I would prefer to first make the small internal change and test it broadly, and then if we're sure it's what we want, document and formalize it.

There is the risk that the first part will happen, but not the second...